sncast_scripts/src/declare_presets/declare_presets.cairo

```diff
@@ -0,0 +1,47 @@
+use openzeppelin_testing::common::IntoBase16String;
+use sncast_std::{declare, get_nonce, FeeSettings, EthFeeSettings};
+
+const MAX_FEE: felt252 = 99999999999999999;
+
+fn main() {
+    let contracts: Array<ByteArray> = array![
+        "AccountUpgradeable",
+        "ERC20Upgradeable",
+        "ERC721Upgradeable",
+        "ERC1155Upgradeable",
+        "EthAccountUpgradeable",
+        "VestingWallet"
+    ];
+
+    let mut consume_latest_nonce = false;
+    let mut nonce = get_nonce('latest');
+
+    for contract in contracts {
+        if (!consume_latest_nonce) {
+            consume_latest_nonce = true;
+        } else {
+            nonce = get_nonce('pending');
+        }
+
+        declare_preset(contract, nonce);
+    };
+
+    println!("");
+}
+
+fn declare_preset(contract: ByteArray, nonce: felt252) {
+    println!("\nDeclaring {contract}:");
```

Comment:
What do we need "\n" for here? Newline between different contracts' declarations?

Reply:
Exactly. I think it's easier to read with the line between each declaration as opposed to a big output cluster